sydent/pipeline.yml

```diff
@@ -1,4 +1,17 @@
 steps:
+  - command:
+      - "echo '--- Install towncrier'"
+      - "pip install towncrier"
+      - "echo '+++ Check newsfile"
+      - "python -m towncrier.check --compare-with=origin/master"
+    label: ":rolled_up_newspaper: Newsfile"
+    plugins:
+      - docker#v3.0.1:
+          image: "python:3.6"
+    branches: "!master !release-*"
+
+  - wait
```

Comment:
I'm a bit unconvinced by this. It's a pain if you have to add a changelog just to check the tests pass on your dev branch.

WDYT?

Reply:
Isn't that the same with Synapse though? The idea here is to avoid using 6 agents when we can know quickly that the CI will need to be re-run for the branch anyway (at least in PRs).

Reply:
it is the same for synapse, but it annoys me there too.

if the CI does get re-run quickly for the branch, the existing build will get cancelled and the agents freed up pretty quickly.

:man_shrugging: Happy to merge this if you like, just wondering if it was an opportunity not to propagate our mistakes.

Reply:
Fair enough, I'll remove it here then